tianshou/data/collector.py

```diff
@@ -321,6 +349,10 @@ def collect(self,
                         (episode_count >= n_episode).all():
                     break
 
+        # finished envs are ready, and can be used for the next collection
+        self._ready_env_ids = np.asarray(
```

Comment:
`np.array` should be faster, and more appropriate IMHO.

Reply:
Good point. I often used ``np.array, np.asarray, np.asanyarray`` interchangeably. I will make it clear the next time:

- If the type is clear (e.g. we want to make an array from a list of int), use ``np.array``.

- If the input has something to do with ndarray (e.g. a list of ndarray), use ``np.asarray``.